Php81.php

```diff
@@ -1,26 +1,15 @@
 <?php
 
-/*
- * This file is part of the Symfony package.
- *
- * (c) Fabien Potencier <fabien@symfony.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
```

Comment:
no way you're allowed to remove copyright headers. you must restore them!

Reply:
I hate copyright headers, such a waste of space. Are they not redundant due to the LICENSE file in the project directory.
But I'll put them back just in case...